### DISCUSSION TITLE
Can you add support vue 3 + ts

### DISCUSSION THREAD

**Discussion by krop24** [umijs/qiankun#2766] [Q&A]
I have a problem with vue project writed in typescript and i didnt know how to resolve this problem. When i use vue without ts it's work but if i use this errors shown
![image](https://github.com/umijs/qiankun/assets/106395139/86074203-e2f9-40f4-8486-66c912f62ed3)

**Comment by aladdin-add** [comment]
vite is not supported by qiankun v2. you can checkout the plugin: https://github.com/tengmaoqing/vite-plugin-qiankun.
